cordapp/src/main/kotlin/net/corda/plugins/Workflow.kt

```diff
@@ -0,0 +1,15 @@
+package net.corda.plugins
+
+import org.gradle.api.tasks.Input
+
+open class Workflow {
+    @get:Input
```

Comment:
Are any of these parameters `@Optional`?

Reply:
There are all optional in as far as the CorDapp developer can specify either a `contract`, `workflow` or `info` `cordapp` section in their respective build.gradle files.

Reply:
Then _technically speaking_ they should also be annotated as `@Optional`. Although this probably doesn't matter unless a Gradle task uses this object as a `@Nested` field.